cmd/drafter-snapshotter/main.go

```diff
@@ -63,6 +63,7 @@ func main() {
 	}
 
 	rawDevices := flag.String("devices", string(defaultDevices), "Devices configuration")
+	ioEngine := flag.String("io-engine", "Async", `Firecracker IO engine (either "Async" or "Sync")`)
```

Comment:
Please make these lowercase for our flag options

Reply:
Resolved in fba812e610826632ed649be89d0d01df7d7b073f